### ISSUE TITLE
No 'zipline/examples' after successful 'conda install -c Quantopian zipline'

### ISSUE THREAD

**Issue by joseortiz3** [quantopian/zipline#1171]
Dear Zipline Community,

I installed zipline via conda, as per instructions. I am looking for the 'examples' folder, but I can assure you there is no 'zipline/examples/' anywhere on my computer.

"You can find other examples in the zipline/examples directory."

I searched the entire computer for 'zipline/', and looked in all of those for 'examples/'. Does not exist. Does conda not install the example folder, like a full installation should?

In other words **can _anyone_ verify that conda install does / does not provide an examples/ directory**

Thank you,
Jose Ortiz
------------------------------------------
* Operating System: Windows 10 64-bit
* Python Version: 2.7.11 | Anaconda 2.5.0 (64-bit)
* Python Bitness: 64-bit
* How did you install Zipline: conda install -c Quantopian zipline
* Python packages: Not relevant
--------------------------------------------

**Comment by richafrank** [comment]
Hi @joseortiz3 ,  Good catch - It looks like the examples directory isn't being included in the conda or even the source distribution.

**Comment by richafrank** [comment]
This should be fixed in zipline 1.0.0.  I see the examples installed in my site-packages now.
